index.d.ts

```diff
@@ -2,14 +2,14 @@
 
 export { getScale as scale, getScale as mode, scales, scales as modes, addChord, getChord as chord, chords, clip, getChordDegrees, getChordsByProgression, progression, arp, midi, Session, max, };
 
-export const getScale: (tonicOctScale: string) => (string | null)[];
+export const getScale: (tonicOctScale: string) => (null | string)[];
```

Comment:
I m curious to know why this order switched, are you using a custom prettier config or something in your editor does this on save maybe?

Reply:
I was confused about this as well as I didn't change anything in the config files. I think it's possible that `npm install` installed a newer version of one of the prettier/linting modules and they changed something there. Looking at package.json it could be tslint, tsling-config-prettier or tslint-plugin-prettier as their versions are prefixed with `^` which means npm will try to grab the latest version.

Either way, I don't think those changes should break anything but please review and let me know if you want it changed.

Reply:
This can remain as is.